uk.co.compucorp.civicrm.tasksassignments/js/src/crm-tasks-workflows/decorators/route.decorator.js

```diff
@@ -45,11 +45,12 @@
      * @return {Function} the function that will resolve the workflow case types.
      */
     function getResolverForWorkflowTypes () {
-      return ['crmApi', function workflowTypesResolver (crmApi) {
-        return crmApi('CaseType', 'get', {
-          category: 'WORKFLOW',
-          options: { limit: 0 }
-        });
+      return ['crmApi', 'customFieldIds', function workflowTypesResolver (crmApi, customFieldIds) {
+        var caseTypeCategoryField = 'custom_' + customFieldIds['caseType.category'];
```

Comment:
Why dont we add the `custom_` prefix, in the backend.

Reply:
That's a CiviCRM thing, that's how custom fields work for them. Apparently in APIv4 you can refer to the field by it's name (`category`), but we use APIv3